examples/step-46/step-46.cc

```diff
@@ -798,8 +799,11 @@ namespace Step46
           stokes_fe_face_values.normal_vector(q);
 
         for (unsigned int k = 0; k < stokes_fe_face_values.dofs_per_cell; ++k)
-          stokes_symgrad_phi_u[k] =
-            stokes_fe_face_values[velocities].symmetric_gradient(k, q);
+          {
+            stokes_symgrad_phi_u[k] =
+              stokes_fe_face_values[velocities].symmetric_gradient(k, q);
+            stokes_phi_p[k] = stokes_fe_face_values[pressure].value(k, q);
```

Comment:
Ugh, did we never initialize this and always just use zero?

Reply:
... even worse. It was passed in non-zero, because the vector was used outside the function.